src/indieweb_utils.py

```diff
@@ -345,28 +345,37 @@ def discover_web_page_feeds(url):
 
     # check for presence of mf2 hfeed
     h_feed = soup.find_all(class_="h-feed")
+    page_title = soup.find("title")
+
+    page_domain = url.split("/")[2]
+
+    valid_mime_types = [
+        "application/rss+xml",
+        "application/atom+xml",
+        "application/rdf+xml",
+        "application/xml",
+        "application/json",
+        "application/mf2+json",
+        "application/atom+xml",
+        "application/feed+json",
+        "application/jf2feed_json",
+    ]
 
-    feeds = []
+    valid_mime_types += user_mime_types
```

Comment:
Since we only iterate over valid_mime_types, we don't need to set store it in a variable here. We can just create our set of mime types when we loop over them. This is easier too when valid_mime_types is define as a set because we can use the `union` method.

```
>>> x = {1,2,3}
>>> y = x.union([3,4,5])
>>> y
{1, 2, 3, 4, 5}
```